jekyll/_cci2/api-intro.md

```diff
@@ -18,9 +18,11 @@ API v2 includes several powerful features (e.g. support for pipelines and pipeli
 
 CircleCI API v1.1 and portions of API v2 are supported and generally available. CircleCI expects to eventually End-Of-Life (EOL) API v1.1 in favor of API v2 as more API v2 endpoints are announced as stable. Further guidance on when CircleCI API v1.1 will be discontinued will be communicated at a future date.
 
-## Introduction to API v2
+## Introduction to API v2 
```

Comment:
JK001/header-jekyll-id-rule Header with custom id rule Expected kramdown ID attribute at next line: 11. You can fix by inserting `{: #introduction-to-api-v2 }` to the next line